VCDMExplainer.md

```diff
@@ -67,6 +67,11 @@ A visual depiction of the verifiable credentials ecosystem is shown below.
   <img src="diagrams/ecosystem.svg" width="100%" height="400">
 </a>
 
+It is worth pointing out why the data model has separate roles of holder and subject. It is recognised that in most cases the holder
```

Comment:
- This doesn't need to say "it is worth pointing out" since the text wouldn't be here otherwise.
- Please adjust line lengths to match the rest of the file.